backend/compact-connect/lambdas/common-python/tests/unit/test_data_model/test_data_client.py

```diff
@@ -0,0 +1,48 @@
+from datetime import date, datetime, timedelta, timezone
+from unittest.mock import MagicMock
+
+from botocore.exceptions import ClientError
+from cc_common.exceptions import CCAwsServiceException
+
+from tests import TstLambdas
+
+
+class TestDataClient(TstLambdas):
+    def test_data_client_deletes_records_if_exception_during_create_privilege_records(self):
```

Comment:
Conceptual question: Which records are you intending to delete if there's an exception?

Reply:
Got it- very helpful!